index.html

```diff
@@ -467,7 +467,7 @@ <h2><dfn>MonetizationState</dfn> enum</h2>
             "<dfn>started</dfn>"
           </dt>
           <dd>
-            Indicates that monetization has started (i.e. the user agent fired the `"monetizationstart"` event).
+            Indicates that monetization has started (i.e. the user agent fired the `"start"` event).
```

Comment:
Yeah, interesting, I believe it may have been the case that the wordier `monetizationstart` etc events were from when the events were emitted on the window object :)

Reply:
Ah, I see... as they now on the `monetization` object, we get those nice shorter ones.